internal-gateway/deployment.yaml

```diff
@@ -37,8 +37,11 @@ spec:
          image: "{{ internal_gateway_image.image }}"
          resources:
            requests:
-             memory: "250M"
-             cpu: "200m"
+             memory: 375Mi
+             cpu: 100m
+           limits:
+             memory: 3750Mi
+             cpu: "1"
```

Comment:
OK, this is slightly complicated and can also impact the router rate limiting.  internal-gateway has (1) request rate limits, and (2) is fixed at 3 replicas (no horizontal scaling) to limit the max incoming batch-driver request rate from workers.  I think 200m was tuned as the observed CPU when we're at the max rate limit.  So, again, this will reduce our max scheduling rate.  Might be safer to leave.

Reply:
ok